investd/reports/overview.py

```diff
@@ -27,13 +28,20 @@
 df_tx = load_transactions()
 df_tx = df_tx[df_tx["timestamp"] <= now]
 
-# %% [markdown]
-# ### Invested amount
+# %%
+
+df_quotes = load_quotes()
+df_portfolio = views.portfolio_value(
+    df_tx, df_quotes, at_date=date(2022, 12, 30)
```

Comment:
hmm why not "now" as at_date?

Reply:
Added instead a `report_date` parameter that can be provided in the command line.